cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -1561,7 +1564,18 @@ void reader::impl::create_global_chunk_info()
                                        schema.type == BYTE_ARRAY and _strings_to_categorical));
     }
 
-    remaining_rows -= row_group_rows;
+    // Adjust for skip_rows when updating the remaining rows after the first group
+    remaining_rows -=
+      (skip_rows) ? std::min<int>(rg.start_row + row_group.num_rows - skip_rows, remaining_rows)
+                  : row_group_rows;
+
+    // Adjust the start_row of the first row group which was left unadjusted during
+    // select_row_groups().
+    if (skip_rows) {
```

Comment:
Not particularly elegant to use skip_rows to make global_skip_rows adjustment to the very first row_group but adds the least code diff here. Suggestions welcome!